LayoutKitTests/OverlayLayoutTests.swift

```diff
@@ -241,6 +241,47 @@ class OverlayLayoutTests: XCTestCase {
         }
     }
 
+    /**
+     Makes sure that the views are ordered correctly - first background, then primary, then overlay.
+     */
+    func testViewOrdering() {
+        let background = SizeLayout(
+            width: Measurement.backgroundWidth,
+            height: Measurement.backgroundHeight,
+            alignment: .topLeading)
+        let overlay = SizeLayout(
+            width: Measurement.overlayWidth,
+            height: Measurement.overlayHeight,
+            alignment: .topLeading)
+        let primary = SizeLayout(
+            width: Measurement.primaryWidth,
+            height: Measurement.primaryHeight,
+            alignment: .topLeading)
+        let layout = OverlayLayout(
+            primary: primary,
+            background: [background, background, background],
+            overlay: [overlay, overlay])
+
+        // Make sure (using the frames) that we have the correct order
```

Comment:
I think this test should call `makeViews` and check via the zorder or hit testing or whatever else is appropriate.